jsonnet-mode.el

```diff
@@ -322,7 +335,61 @@ If not provided, current point is used."
 (defun jsonnet-reformat-buffer ()
   "Reformat entire buffer using the Jsonnet format utility."
   (interactive)
-  (call-process-region (point-min) (point-max) jsonnet-command t t nil "fmt" "-"))
+  (let ((point (point))
+        (file-name (buffer-file-name))
+        (stdout-buffer (get-buffer-create "*jsonnet fmt stdout*"))
+        (stderr-buffer-name "*jsonnet fmt stderr*")
+        (stderr-file (make-temp-file "jsonnet-fmt")))
+    (when-let ((stderr-window (get-buffer-window stderr-buffer-name t)))
+      (quit-window nil stderr-window))
+    (unwind-protect
+        (let* ((only-test buffer-read-only)
```

Comment:
This could be called “only-test-p” instead. It’s been a while since I wrote Lisp regularly.